src/Query/QueryBuilder.php

```diff
@@ -81,6 +81,14 @@ public function setMinimumScore($minScore): QueryBuilder
         return $this;
     }
 
+    public function addSort($field, $order): QueryBuilder
```

Comment:
I just repeated what was done in the other methods.

As a side note, wouldn't `static` instead of `self` achieve the same goal, and allow inheritance ?